test.sh

```diff
@@ -3,10 +3,22 @@
 cd ./BEIMA.Backend.Test
 # Run backend unit tests
 dotnet test ./BEIMA.Backend.Test.csproj || { echo "Unit tests failed"; exit 1; }
-# Change back to root directory
+# Navigate to backend project
+cd ../
+cd ./BEIMA.Backend
+# Start the backend API locally
+func start BEIMA.Backend.csproj --csharp &
+# Switch to FT directory
+cd ../
+cd ./BEIMA.Backend.FT
+# Give enough time for local API to finish starting up
+sleep 15
```

Comment:
Bit hacky, but couldn't think of anything better. open to ideas.

Reply:
When I was working on the cypress testing, I also needed to start the front end server and test. Some of the solutions I found across the web included having a sleep command to wait for the server. I was able to find an npm package that started the server and then ran tests, if there's something like that out there to use on our backend, that would be cool. But for now I think this is fine

Reply:
I've played around with it a bit. When I run it with no wait it eventually fails with a connection refused message.
with the sleep 20 it seems to work ok

with this it fails later (for me), but it isn't related to the project not running:

until pids=$(pidof func start BEIMA.Backend.csproj --csharp &)
do
    sleep 1
done

this is from the first answer when searching for "sh wait for process to start"
There's other results, but all seem to use a loop to wait for the PID or a log message of some type